### PR TITLE
Update 1200 Intro and 1264 weak cipher suites

### PR DESCRIPTION
**Pull request by rusty-snake**
* CBC, SHA1
  * WEAK: security.ssl3.ecdhe_ecdsa_aes_256_sha
  * WEAK: security.ssl3.ecdhe_rsa_aes_256_sha
* Add test from browserleaks with also shows a fingerprint hash and TLS Extensions.

### CODE REVIEW

**Comment by Thorin-Oakenpants** [comment]
We don't **need** a second test. And I don't see why we should add more bad advice/options.

I haven't looked at the two new ciphers you listed, but for the current ones - the chances users have of getting one of these ciphers (outside a downgrade attack = different threat model) is practically zero, since they were deprecated in chrome (and others? safari?) years ago - so I doubt sites are using them.

And, finally, Mozilla look like they are ready to deprecate some of them.

tl;dr: I can't wait to get rid of the cipher section, not add to it.

Please make a compelling case and re-open if you have one

**Comment by earthlng** [comment]
I quite like this proposal. The testpage is nice for the reasons he mentioned, especially the fairly new JA3 fingerprint thing and adding the last 2 "weak" ciphers into the list for completeness sake makes sense too IMO.
I would even add more information namely why each of these ciphers are considered weak like "No PFS, CBC, SHA-1" (same as that test page) and update some of the outdated stuff like "as of FF52".

**Comment by Thorin-Oakenpants** [comment]
> I would even add more information

This as well then? https://github.com/ghacksuserjs/ghacks-user.js/issues/900#issuecomment-597240731  - put some numbers against these

**Comment by earthlng** [comment]
^^ what?

**Comment by Thorin-Oakenpants** [comment]
> ^^ what?

Users need info to make informed decisions. e.g. in the TLS min for years we've said what the risk was (e.g.  `0.5% of TLS web traffic uses 1.0 or 1.1`) so they could balance that up against whether or not to alter their passive fingerprint

I was suggesting we do the same here

**Comment by Thorin-Oakenpants** [comment]
before I forget: the JA3 site I had bookmarked is: https://jwlss.pw/ja3/

**Comment by Thorin-Oakenpants** [comment]
@rusty-snake @earthlng 

OK, then lets get this done. I recommend we close this and do drafts in an issue (I honestly can't work in PRs)
- JA3: add a link (I like mine)
- other test links: pick one, have lots: whatever
- add weaker ciphers
- explain why they're weak (for downgrade attack assessment)
- maybe give some numbers (for breakage assessment)

And when that's done, I'm ready to provide a proper DoH section (if anyone cares) now they've finally ironed out a lot of bugs and added some new UI, and added new prefs. But first things first.

What are your thoughts (on the cipher ideas)? Thumbs up and I'll close this and open an issue with all the info

**Comment by Thorin-Oakenpants** [comment]
no takers?

**Comment by rusty-snake** [comment]
So much to do, so little time. :wink: 
I'm currently at something like this.
```javascript
/* 1261: disable 3DES (effective key size < 128)
 * [1] https://en.wikipedia.org/wiki/3des#Security
 * [2] https://en.wikipedia.org/wiki/Meet-in-the-middle_attack
 * [3] https://www-archive.mozilla.org/projects/security/pki/nss/ssl/fips-ssl-ciphersuites.html ***/
   // user_pref("security.ssl3.rsa_des_ede3_sha", false);

/* 0000: Disable weak CIPHERS
   128: 
   SHA1:
   CBC:
   NOPFS:
   DHE:
***/
   // user_pref("security.ssl3.ecdhe_ecdsa_aes_128_sha", false); // 128; SHA1; CBC;
   // user_pref("security.ssl3.ecdhe_rsa_aes_128_sha", false); // 128; SHA1; CBC;
   // user_pref("security.ssl3.dhe_rsa_aes_128_sha", false); // 128; SHA1; CBC; DHE;
   // user_pref("security.ssl3.dhe_rsa_aes_256_sha", false); // SHA1; CBC; DHE;
   // user_pref("security.ssl3.rsa_aes_128_sha", false); // 128; SHA1; CBC; NOFPS;
   // user_pref("security.ssl3.rsa_aes_256_sha", false); // SHA1; CBC; NOFPS;
   // user_pref("security.ssl3.ecdhe_ecdsa_aes_256_sha", false); // SHA1; CBC;
   // user_pref("security.ssl3.ecdhe_rsa_aes_256_sha", false); // SHA1; CBC;
```

**Comment by Thorin-Oakenpants** [comment]
OK. This is a lot easier to nut out in an issue

**Comment by Thorin-Oakenpants** [comment]
I can't deal with this conflict shit from v74: I'll post a new commit
